common/protos/common/events.proto

```diff
@@ -68,4 +69,16 @@ message EventPublication {
     ContractTransaction ctx = 1;
     string app_url = 2;
   }
-}
\ No newline at end of file
+}
+
+message EventStates {
+    repeated EventState states = 1;
+}
+
+// the payload that is used for the communication between the requesting relay
+// and its network
+message EventState {
+  common.state.RequestState state = 1;
```

Comment:
Whenever we use, It will be like `EventState.state` or `event_state.state`, if you strongly feel to change this name, I can change it (just will have to change it from multiple places within relay code).

Reply:
Looks fine to me, no need of change I guess.